src/js/themes/vanilla.js

```diff
@@ -524,6 +520,8 @@ export default deepFreeze({
     },
   },
   select: {
+    backgroundColor: '#ffffff',
```

Comment:
Why is this needed in select? I would expect that the drop.backgroundColor would be sufficient.

Reply:
yeah let's start with the drop background color! good point!

Reply:
I retract my suggestion to tie to drop, based on your clarifications. Can we name this just `background`, per my previous comments?